apps/prairielearn/src/server.js

```diff
@@ -367,9 +367,13 @@ module.exports.initExpress = function () {
   // assets system with cacheable assets.
   app.use(express.static(path.join(APP_ROOT_PATH, 'public')));
 
-  // This is included for backwards-compatibility with pages that might still
-  // expect to be able to load files from the `/node_modules` route.
-  app.use('/node_modules', staticNodeModules('.'));
+  // For backwards compatibility, we redirect requests for the old `node_modules`
+  // route to the new `cacheable_node_modules` route.
+  app.use('/node_modules', (req, res) => {
+    // Strip the leading slash.
+    const assetPath = req.url.slice(1);
```

Comment:
Just to check, will `req.url` already have the `/node_modules` stripped? We don't want `node_modules` in `assetPath`, I think?

Reply:
Yeah, `req.url` excludes the route part of the path (`/node_modules`, in this case). `req.originalUrl` contains the full one, including `/node_modules`. This is documented here: https://expressjs.com/en/api.html#req.originalUrl